src/Items/ItemBucket.h

```diff
@@ -164,13 +166,7 @@ class cItemBucketHandler :
 				// Plugin disagrees with the washing-away
 				return false;
 			}
-
-			cBlockHandler * Handler = BlockHandler(CurrentBlockType);
-			if (Handler->DoesDropOnUnsuitable())
-			{
-				cChunkInterface ChunkInterface(a_World->GetChunkMap());
-				Handler->DropBlock(ChunkInterface, *a_World, a_PluginInterface, a_Player, BlockPos.x, BlockPos.y, BlockPos.z);
-			}
+			a_World->DropBlockAsPickups(BlockPos, a_Player, nullptr);
```

Comment:
Why do you ignore `DoesDropOnUnsuitable` now?

Reply:
Makes more sense - the block needs to go in any way, so drop it as nature-produced pickup - which is exactly what `DropBlockAsPickup` with nullptr Digger and Tool promise.